### ISSUE TITLE
tkt-maintenance-public.js

### ISSUE THREAD

**Issue by joyously** [TukuToi/tukutoi-maintenance#19]
https://github.com/TukuToi/tkt-maintenance/blob/18c552f0089296be71e39d1c9521a61df0f6d6e1/public/js/tkt-maintenance-public.js#L61

If the timer expires, shouldn't the site be shown?
This text is not translated.

**Comment by smileBeda** [comment]
No, that is unsafe. It might be the user entered a wrong time, just by accident, and thus, going in and deciding for the user that now its time to show the site again, would be intrusion.

Also, the timer is location based and does not really consider timezones, it could produce unexpected results.

It is more of a "nice" visual feedback for a webmaster to show wether a update is about to be 10 hours or perhaps 7 days, who knows.

The text not only needs to be translated but likely also needs to be customisable, rather than hardcoded
Good catch.
Putting this on the list ..
